style.css

```diff
@@ -344,3 +373,137 @@ form {
   background-repeat: no-repeat;
   background-position: top 1em left 18.5em, top 1em left 16.5em, bottom 5em left 14em, top 1em left 10em, bottom 5em left 14em;
 }
+
+@media only screen and (min-width: 768px) {
+  .wrap {
+    width: 100%;
+  }
+
+  header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 48px;
+    width: 100%;
+  }
+
+  nav {
+    width: 100%;
+  }
+
+  .menu-cont {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 48px;
+    width: 100%;
+    border: 1px solid white;
+    padding-left: 7%;
+  }
+
+  .dsktp-lf-menu {
+    display: block;
+  }
+
+  .menu-icon {
+    display: none;
+  }
+
+  .dst-cont {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 48px;
+    width: 100%;
+  }
+
+  .menu-ul {
+    display: flex;
+    width: 400px;
+    border: 1px solid white;
+  }
+
+  .menu-ul li {
+    list-style: none;
+    margin: 10px;
+  }
+
+  .head_description {
+    width: 100%;
+    padding: 15em;
+    background-image: url("images/desktop/desktop_image/header_bg.svg");
+    background-color: white;
+  }
+
+  .hero-area {
+    margin: 1px;
+  }
+
+  .all_middle_container {
+    width: 80%;
+    margin: auto;
+    align-items: center;
+    padding: 2em;
+    border: 1px solid white;
+  }
+
+  .mobile_project {
+    display: none;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    padding: 16px;
+    gap: 12px;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    background-color: white;
+  }
+
+  .right-block {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    padding-left: 8%;
+    padding-top: 4%;
+  }
+
+  .left-block {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    padding-left: 8%;
+    padding-top: 4%;
+  }
+
+  .about-container {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 37px;
+  }
+
+  .language-list {
+    display: flex;
+    flex-direction: row;
+    gap: 8px;
+    padding: 1em;
+  }
+
+  .second-card {
+    width: 100%;
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .multi-post-stories {
+    display: flex;
+    flex-direction: row;
+  }
+}
+
+@media only screen and (max-width: 768px) {
+  .desktop_project {
+    display: none;
+  }
+}
```

Comment:
- Kindly remove this media query, or you can add it before the media query for large screen sizes.  The mobile-first approach design that you implemented works based on the screen sizes small then large :+1:

Reply:
Moved